File: python/간단한 식 계산하기.py
def solution(binomial):
    question = binomial.split(" ")
    a = int(question[0])
    b = question[1]
    c = int(question[2])
    
    if(b == '+'):
        return a + c
    elif(b == '-'):
        return a - c
    elif(b == '*'):
        return a * c

File: javascript/간단한 식 계산하기.js
const solution = (binomial) => {
    const s = binomial.split(" ");
    const op = s[1];
    const n1 = parseInt(s[0]);
    const n2 = parseInt(s[2]);
    
    if(op == "+") {
        return n1 + n2;
    }else if(op == "-"){
        return n1 - n2;
    }else {
        return n1 * n2;
    }
}